invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/search/SearchResultItemLayout.js

```diff
@@ -45,18 +45,27 @@ class SearchResultItemComponent extends Component {
         <Table.Cell key={`user-name-${result.id}`} data-label={i18next.t("Username")}>
           @{result.username}
           {result.identities.orcid && (
-            <img
-              className="inline-id-icon ml-5"
-              src="/static/images/orcid.svg"
-              alt="ORCID"
-            />
+            <a href={`https://orcid.org/${result.identities.orcid}`} rel="noreferrer">
+              <img
+                className="inline-id-icon ml-5"
+                src="/static/images/orcid.svg"
+                alt="ORCID"
+              />
+            </a>
+          )}
+          {result.identities.github && (
+            <a
+              href={`https://api.github.com/user/${result.identities.github}`}
```

Comment:
Is it correct to go to the API endpoint?
```suggestion
              href={`https://github.com/${result.identities.github}`}
```

Reply:
yep, going to https://api.github.com/user/6676843 works but https://github.com/6676843 does not